<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NSController</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="devicelist">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="buddy">
     <cstring>devicelist</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Connect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Refresh">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Disconnect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
